FreeRTOS/FreeRTOS
# Add x64 configuration to Win32-MSVC demo

Description
-----------
Add x64 configuration to Win32-MSVC demo.

This was originally contributed in this PR - https://github.com/FreeRTOS/FreeRTOS/pull/1139

Test Steps
-----------
Tested locally for both Win32 and x64.

Checklist:
----------



Related Issue
-----------
https://github.com/FreeRTOS/FreeRTOS/pull/1139


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
